things/policies/api/grpc/client.go

```diff
@@ -67,32 +67,32 @@ func (client grpcClient) Authorize(ctx context.Context, req *policies.AuthorizeR
 	return &policies.AuthorizeRes{ThingID: ares.thingID, Authorized: ares.authorized}, nil
 }
 
-func (client grpcClient) Identify(ctx context.Context, req *policies.Key, _ ...grpc.CallOption) (*policies.ClientID, error) {
+func (client grpcClient) Identify(ctx context.Context, req *policies.IdentifyReq, _ ...grpc.CallOption) (*policies.IdentifyRes, error) {
 	ctx, cancel := context.WithTimeout(ctx, client.timeout)
 	defer cancel()
 
-	res, err := client.identify(ctx, identifyReq{key: req.GetValue()})
+	res, err := client.identify(ctx, identifyReq{secret: req.GetSecret()})
 	if err != nil {
 		return nil, err
 	}
 
-	ir := res.(identityRes)
-	return &policies.ClientID{Value: ir.id}, nil
+	ires := res.(identityRes)
+	return &policies.IdentifyRes{Id: ires.id}, nil
```

Comment:
```suggestion
	return &policies.IdentifyRes{Id: res.(identityRes).id}, nil
```

Reply:
I have left this to maintain uniformity with other methods